src/crewai/crew.py

```diff
@@ -127,7 +127,7 @@ class Crew(BaseModel):
         description="An Instance of the EntityMemory to be used by the Crew",
     )
     embedder: Optional[dict] = Field(
-        default={"provider": "openai"},
```

Comment:
_Question_: Is this right? Usually the default embedder is Openai, setting to None will not break stuff?